app/service/ScalaTestRunner.scala

```diff
@@ -14,31 +14,71 @@ object ScalaTestRunner {
   val failedMarker = "FAILED"
   val failedInRuntimeMarker = "failed in runtime"
   val userClass = "UserSolution"
+  val classDefPattern = "class\\s*([\\w\\$]*)".r
+  val traitDefPattern = "trait\\s*([\\w\\$]*)".r
+  val defaultImports = "import org.scalatest._"
 
+  import scala.reflect.runtime._
+  val cm = universe.runtimeMirror(getClass.getClassLoader)
+  import scala.tools.reflect.ToolBox
+  val tb = cm.mkToolBox()
+
+  /**
+    * Runs suite loaded in runtime with dynamic solution
+    */
   def execSuite(solution: String, suiteClass: Class[Suite], solutionTrait: Class[AnyRef]): String = {
-    Try {
+    def execution() = {
```

Comment:
It is used for calling the tryExecSuite by name, not by value. Can you propose another solution?